app/javascript/utils/utils.js

```diff
@@ -8,3 +8,8 @@ export default {
   cleanField,
   SALESFORCE_DATE_FORMAT,
 }
+
+
+export const formatPercent = (value) => {
```

Comment:
about the name of the file 'utils/utils` we should a TODO to fix this name not sure if it's says much

Reply:
yeah we have a lot of util files, I'm not sure what goes where.